data/coalition missions.txt

```diff
@@ -108,6 +109,7 @@ mission "Coalition: Contributor"
 		government "Coalition"
 	destination "Ring of Friendship"
 	on offer
+		log "After doing a few dozen jobs for the Coalition, they're finally wiling to let me purchase some of their technology. Doesn't sound like they'll sell me any of their weapons, though."
```

Comment:
It might be clearer to say something like, "Got permission from the Coalition to purchase their technology. But, it turns out that their weapons and other military-grade outfits are only available to the Heliarchs, not to civilians."

(It's important that *no one* in the Coalition is allowed to have weapons on their ships except the Heliarchs; it's not just that they won't sell weapons to you because you're an alien.)